### DISCUSSION TITLE
Add some screenshots on readme page

### DISCUSSION THREAD

**Discussion by Cyberavater** [Open-Shell/Open-Shell-Menu#1106] [General]
I'd like to know more about this project before trying it out. Can you be kind enough to post some screenshots on the read me page? Thanks!

**Comment by blackcrack** [comment]
I hope this is enough..
[Classic Open-Shell Almanach.zip](https://github.com/Open-Shell/Open-Shell-Menu/files/9386366/Classic.Open-Shell.Almanach.zip)
this have i make by self, yesterday.. a bit luck for you, you ask now for it..
It's a odt (Open Document Text), open with Libre Office and exportable as pdf and so on..
have fun by read :)
[4.4.173 Downloaded direct by ci.appveyor.com finally **(Inoffizial Release)** ](https://github.com/Open-Shell/Open-Shell-Menu/discussions/1109)
